src/components/editor/markdown-renderer/markdown-renderer.scss

```diff
@@ -1,13 +1,26 @@
 @import '../../../../node_modules/github-markdown-css/github-markdown.css';
 
 .markdown-body {
-  max-width: 758px;
   font-family: 'Source Sans Pro', "twemoji", sans-serif;
 
   .alert > p, .alert > ul {
     margin-bottom: 0;
   }
 
+  // I know this is bad :( sry
```

Comment:
nobody wondered about this?

Reply:
no we saw that, but we didn't find a good solution for this. We would basically need to enumerate all possible dom elements that could be rendered in the markdown-renderer at top level. Since we support HTML in our Markdown Files that's everything…

Reply:
that's the more helpful comment right there.
I would propose putting that sentence in there, it will make sense in a year or a decade from now. And people won't feel bad about it :-)